pyproject.toml

```diff
@@ -7,12 +7,9 @@ requires-python = ">=3.12"
 dependencies = ["docker>=7.1.0", "mcp>=1.1.0", "pydantic-settings>=2.6.1"]
 license = { file = "LICENSE" }
 keywords = ["docker", "mcp", "server"]
-
-requires-python = ">=3.12"
```

Comment:
This is required for publishing to PyPi, please revert the changes you've made to this file. I'm open to removing the `.python-version` file if you'd like to do that

Reply:
@ckreiling It is already declared in line 6.